prov/gni/src/gnix_nic.c

```diff
@@ -359,12 +355,8 @@ static int __process_rx_cqe(struct gnix_nic *nic, gni_cq_entry_t cqe)
 			GNIX_DEBUG(FI_LOG_EP_DATA,
 				  "Processing VC RX (%p)\n",
 				  vc);
-			ret = _gnix_vc_dequeue_smsg(vc);
-			if (ret != FI_SUCCESS) {
-				GNIX_WARN(FI_LOG_EP_DATA,
-					"_gnix_vc_dqueue_smsg returned %d\n",
-					ret);
-			}
+			ret = _gnix_vc_rx_schedule(vc);
+			assert(ret == FI_SUCCESS);
```

Comment:
did you remove the warning statement because ```_gnix_vc_rx_schedule``` is always suppose to return FI_SUCCESS?